at_server_spec/lib/src/connection/at_connection.dart

```diff
@@ -31,4 +31,7 @@ abstract class AtConnectionMetaData {
   bool isPolAuthenticated = false;
   bool isStream = false;
   String? streamId;
+  /// Represents the version of the client initiated the connection.
+  /// Defaults to latest which indicates the latest release.
+  String clientVersion = 'latest';
```

Comment:
Can't default to 'latest'. By definition the older a client is, the less likely it is to be sending its version. Suggest default to 'n/a' (not available)

Reply:
I've made this change and pushed to the branch